fdbserver/BlobWorker.actor.cpp

```diff
@@ -3234,6 +3234,7 @@ ACTOR Future<Void> waitForVersion(Reference<GranuleMetadata> metadata, Version v
 
 	// wait for change feed version to catch up to ensure we have all data
 	if (metadata->activeCFData.get()->getVersion() < v) {
+		// FIXME: add future version timeout and throw here, same as SS
```

Comment:
I ended up not doing this because there's only 1 blob worker for each granule, so there's little benefit to throwing and retrying vs just waiting.